vite.config.ts

```diff
@@ -81,11 +81,14 @@ export default defineConfig(({ mode }) => ({
   },
   plugins: [
     react({
-      plugins: [
-        // https://jotai.org/docs/tools/swc
-        ['@swc-jotai/react-refresh', {}],
-        ['@swc-jotai/debug-label', {}],
-      ],
+      plugins:
+        mode !== 'production'
+          ? [
+              // https://jotai.org/docs/tools/swc
+              ['@swc-jotai/react-refresh', {}],
+              ['@swc-jotai/debug-label', {}],
+            ]
+          : [],
```

Comment:
Awesome, I was implementing exactly this change to avoid production deployment complains!
Thank you!